css-fonts-4/Overview.bs

```diff
@@ -8054,10 +8054,32 @@ The <code id="cssfontfacerule2">CSSFontFaceRule</code> interface</h3>
 
 The <dfn id="cssfontfacerule-interface">CSSFontFaceRule</dfn> interface represents a <<@font-face>> rule.
 
+<!--
+	FIXME: Bikeshed doesn't like the dashed attributes on CSSFontFaceDescriptors.
+-->
+
 <pre class="idl">
+[Exposed=Window]
+interface CSSFontFaceDescriptors : CSSStyleDeclaration {
+	attribute [LegacyNullToEmptyString] CSSOMString src;
+	attribute [LegacyNullToEmptyString] CSSOMString fontFamily;
+	attribute [LegacyNullToEmptyString] CSSOMString fontStyle;
+	attribute [LegacyNullToEmptyString] CSSOMString fontWeight;
+	attribute [LegacyNullToEmptyString] CSSOMString fontStretch;
```

Comment:
`font-stretch` (descriptor) is now a legacy alias of `font-size` (https://github.com/w3c/csswg-drafts/commit/b0660ee070c55cc9b2669498460fc0210d105403). I guess `fontSize` should also be defined.